### ISSUE TITLE
HTML element with 'progress' class becomes hidden during document loading

### ISSUE THREAD

**Issue by t-mog** [stephanrauh/ngx-extended-pdf-viewer#124]
**Describe the bug**
An HTML element, that is a parent to an element with `.progress` class, and is on the same page with PDF viewer, becomes hidden during the PDF document loading.
Probably related to the fix of the bug https://github.com/stephanrauh/ngx-extended-pdf-viewer/issues/21.

**Version info**
- Version of ngx-extended-pdf-viewer: 0.9.55
- Version of the JavaScript files pdf.js, pdf-worker.js and viewer.js: default files

**To Reproduce**
```html
<div>
  This will be hidden
  <div class="progress"></div>
</div>
<ngx-extended-pdf-viewer src="path" useBrowserLocale="true"></ngx-extended-pdf-viewer>
```

**Comment by stephanrauh** [comment]
I suppose that's correct. You'll have to update the file `viewer.js` when I've fixed the bug. Here's the culprit:
<img width="492" alt="image" src="https://user-images.githubusercontent.com/3045767/63192019-4c427180-c06a-11e9-9c59-426b1f04310f.png">
Adding insult to injury, I suppose the wrong `.progress` div is shown and hidden. The query finds your progress bar instead the the progress indicator of the PDF viewer. :)

**Comment by stephanrauh** [comment]
Before I forget: Thanks for the reproducer!

**Comment by stephanrauh** [comment]
I've just published the bug fix in version 1.0.1.

**Comment by t-mog** [comment]
Thanks for such a quick fix!
